client/lib/signup/actions.js

```diff
@@ -53,7 +53,15 @@ const SignupActions = {
 	},
 
 	processedSignupStep( step, errors, providedDependencies ) {
-		analytics.tracks.recordEvent( 'calypso_signup_actions_complete_step', { step: step.stepName } );
+		const { stepName } = step;
+
+		analytics.tracks.recordEvent( 'calypso_signup_actions_complete_step', { step: stepName } );
+
+		if ( stepName === 'survey-user' ) {
```

Comment:
Thanks for the suggestion, I didn't know we have these `stepActions`. the `signup/` code is huge. :)